ansible_navigator/image_manager/__init__.py

```diff
@@ -1,3 +1,4 @@
 """ image manager """
 
 from .image_puller import ImagePuller
```

Comment:
Similar to previour comment `image_puller` can be renamed to `pull.py`?

Reply:
yes

Reply:
yes